src/seedu/addressbook/data/person/Address.java

```diff
@@ -15,6 +17,11 @@
     public final String value;
     private boolean isPrivate;
 
+    public Block block;
```

Comment:
Change the accessibility of these variables private, according to coding standards.